tests/integration-all/api-gateway/service/serverless.yml

```diff
@@ -57,6 +57,15 @@ functions:
           path: api-keys
           method: GET
           private: true
+          minimal:
+  timeout:
+    handler: core.timeout
+    timeout: 1
+    events:
+      - http:
+          method: GET
+          integration: lambda
+          path: integration-lambda-timeout
```

Comment:
Could we simplify that path? Something like `timeout` or `lambda-timeout` should be sufficient since we're operating in an integration test setup here.

Reply:
Here the `integration` token is to reflect that we're using the  _integration: lambda_ (not _integration: proxy_ which is default) for an API endpoint (it's not to indicate that it's an _integration_ test, as that indeed is obvious)

Unfortunately both _integration_ and _lambda_ terms can be confusing, as seem very generic.

Reply:
I see. That makes sense :+1:

Thanks for clarifying!